### PR TITLE
Duplicate of previous PR

### PR DESCRIPTION
**Pull request by theofficetroll**
Previous fork became disconnected.

### CODE REVIEW

**Comment by theofficetroll** [comment]
https://trello.com/c/Neh5Z5E2

**Review by gektin — APPROVED** [review]
These changes are responsive to my comments. Only a small nit left (and not hanging up approval for it), but if you look at database/index.js at its final line and database/seed.js in its, you'll see that one has a new line at the end of the file, and the other doesn't. I believe the airbnb style guide recommends including the new line at the end of a file, but either way, pick one and stick with it throughout your code.

**Comment by theofficetroll** [comment]
Added additional skeleton for server, react, webpack, jest, and the kitchen sink.

https://trello.com/c/9mdN4Svc

File: src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  render() {
+    return (
+      <div>
+        WIP
+      </div>
+    )
+  }
+};
+
+ReactDOM.render(<Navbar />, document.getElementById('Navbar'));
```
Comment (redperezz) [inline src/index.jsx:22]: [WIP] ✓

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return Navbar.insertMany(navbarData)
+  .then (result => {
+    console.log ('saved to database');
+  })
+  .catch (err => console.log ('failed to save to database', err));
+};
+
+module.exports = {
+  navbarSave, Navbar
+}
```
Comment (redperezz) [inline database/index.js:41]: Very nice to seed your data when db connected! ✓

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      bundleId,
+      countDown
+    })
+  }
+  return timerData;
+};
+
+module.exports = {
+  seedGen
+}
```
Comment (redperezz) [inline database/seed.js:20]: Not sure if I'm seeing the whole file but not sure if you're using the require on line 1. 

Took me a minute to understand how you're using the returned array from seedGen in navbarSave but makes sense now that I see that it's an array of objects, great job! ✓

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <head>
+    <title>HumbleBundle</title>
+  </head>
+  <body>
+    <div id="Navbar">
+      Navbar
+    </div>
+    <script src="./static/js/bundle.js"></script>
+  </body>
+</html>
```
Comment (redperezz) [inline dist/index.html:12]: ✓

File: server/index.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist'));
+
+const PORT = 3102;
+
+app.listen(PORT, () => console.log('listening on port', PORT));
```
Comment (redperezz) [inline server/index.jsx:13]: ✓

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
+        exclude: /node_modules/,
+        query: {
+          presets: ['react']
+        }
+      }
+    ]
+  }
+};
```
Comment (redperezz) [inline webpack.config.js:18]: ✓

**Review by redperezz — APPROVED** [review]
Very nice setup! Looking forward to when you make the countdown more granular!

File: database/seed.js (hunk 3 above)
Comment (theofficetroll) [reply database/seed.js:20]: It's a little overkill with only 2 values, but it's a really convenient way to save the data! Came across it doing my MVP.

I have commented out line 1 for the next commit. I had originally had the seeding separated differently between the files.

**Pull request merged**